openupgrade_scripts/scripts/fleet/18.0.0.1/upgrade_analysis_work.txt

```diff
@@ -0,0 +1,64 @@
+---Models in module 'fleet'---
+new model fleet.vehicle.send.mail [transient]
+# NOTHING TO DO
+---Fields in module 'fleet'---
+fleet        / fleet.vehicle            / power_unit (selection)        : NEW required, selection_keys: ['horsepower', 'power'], hasdefault: default
+fleet        / fleet.vehicle            / vehicle_range (integer)       : NEW
+fleet        / fleet.vehicle.model      / activity_ids (one2many)       : NEW relation: mail.activity
+fleet        / fleet.vehicle.model      / message_follower_ids (one2many): NEW relation: mail.followers
+fleet        / fleet.vehicle.model      / message_ids (one2many)        : NEW relation: mail.message
+fleet        / fleet.vehicle.model      / power_unit (selection)        : NEW required, selection_keys: ['horsepower', 'power'], hasdefault: default
+fleet        / fleet.vehicle.model      / vehicle_range (integer)       : NEW
+# DONE set power_unit to horsepower default is power) if horsepower is > 0. Other fields are new with no consequence on existing data
+---XML records in module 'fleet'---
```

Comment:
You have to separate lines for better reading, and the syntax is `DONE: ` + `pre-migration/post-migration/end-migration` + `: `.

Reply:
will do :)